blueocean-dashboard/src/main/js/AllPipelines.jsx

```diff
@@ -0,0 +1,14 @@
+import React, { Component } from 'react';
+import OrganisationPipelines from './OrganisationPipelines';
+
+class AllPipelines extends Component {
+
+    render() {
+        return (
+            <OrganisationPipelines { ... this.props } />
+        );
+    }
+
+}
+
+export default AllPipelines;
```

Comment:
Tried an experiment with using higher-order components to wrap our old `OrganisationPipelines` and pass in organization (see: `OrganizationPipelines`). As I worked through it, these components became simpler and simpler, to the point where they may not even be necessary. (This one clearly isn't, so should probably nuke it and use the original directly).